src/config.json

```diff
@@ -963,6 +963,78 @@
     "ipfs.io"
   ],
   "blacklist": [
+    "1inch-drop.org",
+    "1inch-event.top",
+    "500coin-get.top",
+    "account-coinbase.info",
+    "account-page-coinbase.com",
+    "amazon802.work.gd",
+    "apecoinbase.xyz",
+    "auth-account-coinbase.com",
+    "briddgemetis.com",
+    "claim500crypto.top",
+    "claimcryptoeth.xyz",
+    "coinbase-com.aljadiriyah.com",
+    "coinbase-com.bienestarencolombia.com",
+    "coinbase-com.domenicorizzitelli.com",
+    "coinbase-com.house-cleaning-boca-raton.com",
+    "coinbase-com.matinumampimpa.com",
+    "coinbase-com.randieslist.com",
+    "coinbase-com.smartcars-dubai.com",
+    "coinbase-helpsupport.com",
+    "coinbase-report.parliamentary.live",
+    "coinbase-reportsc.weyas.live",
+    "coinbase-servapp.beenurajpootfilms.com",
+    "coinbase-support.participating.me",
+    "coinbase.20biz.com",
+    "coinbase.cryptocurrencysupport.org",
+    "coinbase.internetagentur.com",
+    "coinbase.login-account-support.com",
+    "coinbase.login.stickerprinting.sg",
+    "coinbase.myzone2fa.com",
+    "coinbase.reset-account-support.com",
+    "conect-metamask.com",
+    "connectiongeneral.com",
+    "dappsfortune.pages.dev",
+    "dex-air.top",
+    "ethereum-bal.com",
+    "ethereum-stake.top",
+    "ethereumclassic.com.cn",
+    "ethereumfunding.com",
+    "exclaim-inc.info",
+    "https-web3-1inch.io",
+    "keeper-wallet.app",
+    "launchpad-apps.network",
+    "metamasck.dyn.ddnss.de",
+    "metamash.io",
+    "metamask-protectwallet.com",
+    "metamask-support-connect.com",
+    "metamaska.site",
+    "metamaskdev.com",
+    "metamast.com",
+    "mr-zkazino.site",
+    "musk.exchange",
+    "pancakeswap.globalsoftwaresupport.com",
+    "pancakeswap.online",
+    "pancakeswapairdrop.net",
+    "pancakeswapp.fans",
+    "pancakeswapper.com",
+    "reset-page-coinbase.com",
+    "resssetpassword-onlycoinbase3.com",
+    "resssetpassword-onlycoinbase4.com",
+    "start-seedify.com",
+    "tocx.net",
+    "trustwallet.danosglobalbank.com",
+    "uniswap.org.ru",
+    "uniswap.v2-app.org",
+    "uo-coinbase.xyz",
+    "verify-page-coinbase.com",
+    "walletcryptomixer.com",
+    "walletmvalidator.online",
+    "web3-defi-connect.pages.dev",
+    "winner-crypto.top",
+    "xearn.pro",
```

Comment:
Hey, team,

I am the organizer of https://xearn.pro/, the website has been running smoothly for a year and working for 300 users currently, not sure why the website is on the block list, which has **affected our reputation seriously**, is there a detailed security report for us to feedback and modify.

Thank you.